mk/e2e.new.mk

```diff
@@ -23,6 +23,11 @@ E2E_UNIVERSAL_BIN_DEPS+= build/kumactl
 E2E_ENV_VARS+= GINKGO_EDITOR_INTEGRATION=true
 endif
 
+# List of images that we pull from the Internet in the E2E tests.
+# This list is a best effort, so we can pull and preload images to Kube clusters. In case of network error, we can fail fast.
+# In the future, we can cache those images so we won't pull this from external sources every single time.
+EXTERNAL_TEST_IMAGES = bitnami/kubectl:1.27.5 mendhak/http-https-echo:23 otel/opentelemetry-collector-contrib:0.92.0
```

Comment:
We should really get rid of: mendhak/http-https-echo I'm pretty sure our test image does this just fine

Reply:
https://github.com/kumahq/kuma/issues/9376